MapboxNavigation.podspec

```diff
@@ -44,7 +44,7 @@ Pod::Spec.new do |s|
   s.module_name = "MapboxNavigation"
 
   s.dependency "MapboxCoreNavigation", "#{s.version.to_s}"
-  s.dependency "Mapbox-iOS-SDK", "~> 3.6"
+  s.dependency "Mapbox-iOS-SDK", "4.0.0-beta.1"
```

Comment:
Remember to update the Cartfile and MapboxNavigation-Documentation.podspec before merging.